restapi/src/main/java/io/stargate/web/docsapi/service/QueryExecutor.java

```diff
@@ -116,20 +225,42 @@ public QueryExecutor(DataStore dataStore) {
     }
   }
 
-  private Iterable<Accumulator> seeds(
-      BoundQuery query, ResultSet rs, List<Column> keyColumns, ExecutionContext context) {
+  private Accumulator toSeed(
+      DocProperty property, Comparator<DocProperty> comparator, List<Column> keyColumns) {
+    Row row = property.row();
+    String id = row.getString("key");
```

Comment:
nit: I think we should stop having the column names used by the DocsAPI directly as the literal, wanna use the constant value from the `io.stargate.web.docsapi.service.query.QueryConstants`?

Reply:
good point. will fix :+1:

Reply:
fixed in ee5522221621015605a3fab3b196e37452a7843d